pkg/workbench/service/workbench-service.go

```diff
@@ -150,16 +153,16 @@ func (s *WorkbenchService) getProxy(proxyID proxyID) (*proxy, error) {
 	targetURL, err := url.Parse(xpraUrl)
 	if err != nil {
 		return nil, fmt.Errorf("Failed to parse url: %w", err)
-
 	}
 
+	reg := regexp.MustCompile(`^/api/rest/v1/workbenchs/[0-9]+/stream`)
```

Comment:
you could even compile that regexp once rather than for each call made to `getProxy`.